ragna/__init__.py

```diff
@@ -14,3 +14,12 @@
 
 from . import assistants, core, deploy, source_storages
 from .core import Rag
+
```

Comment:
nitpick: why not include `__version__` here?

Reply:
Oversight. Good catch!